cloud/scope/machine.go

```diff
@@ -997,6 +997,10 @@ func (m *MachineScope) getLaunchVolumeAttachments() []core.LaunchAttachVolumeDet
 	for _, attachment := range volumeAttachmentsInSpec {
 		if attachment.Type == infrastructurev1beta2.IscsiType {
 			volumes = append(volumes, getIscsiVolumeAttachment(attachment.IscsiAttachment))
+		} else if attachment.Type == infrastructurev1beta2.ParavirtualizedType {
+			volumes = append(volumes, getParavirtualizedVolumeAttachment(attachment.ParavirtualizedAttachment))
+		} else {
+			m.Logger.Info("Block Volume attachment type not supported")
```

Comment:
Lets change this to "Unknown attachment type not supported" or something like that